op-challenger/init_game

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# build the challenger to keep it up to date
+make
```

Comment:
We probably shouldn't assume we're run from within the `op-challenger` dir (ie lots of people will run `./op-challenger/init_game`).
```suggestion
DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
cd "$DIR"

# build the challenger to keep it up to date
make
```

Can use that same construct in all the scripts to make them simpler to use.